addons/website_slides/static/src/tests/activity_tests.js

```diff
@@ -0,0 +1,111 @@
+odoo.define('website_slides/static/src/tests/activity_tests.js', function (require) {
+'use strict';
+
+const components = {
+    Activity: require('mail/static/src/components/activity/activity.js'),
+};
+
+const {
+    afterEach,
+    beforeEach,
+    start,
+} = require('mail/static/src/utils/test_utils.js');
+
+QUnit.module('website_slides', {}, function () {
+QUnit.module('static', {}, function () {
+QUnit.module('src', {}, function () {
+QUnit.module('tests', {}, function () {
```

Comment:
sorry I wasn't clear: we omit those 3 in qunit module definition because they don't add any information (static, src, tests)